src/clj/commiteth/middleware.clj

```diff
@@ -64,15 +69,22 @@
         (wrap-authentication backend)
         (wrap-authorization backend))))
 
+(defstate session-cleaner
+  :start #(start-cleaner *db*)
+  :stop  #(stop-cleaner session-cleaner))
+
+(def redirect-to-home-page (response/redirect "/"))
```

Comment:
No need for a top-level def here I'd say, just pass `(response/redirect "/")` directly.